### ISSUE TITLE
Width: 100%

### ISSUE THREAD

**Issue by errorx666** [zefoy/ngx-color-picker#55]
I want the dialog to be the same width as the textbox.

I have managed to get it to work like this:
```html
<input
	type="text"
	#color
	cpPosition="bottom"
	[cpWidth]="color.offsetWidth"
	[(colorPicker)]="color"
	[value]="color"
/>
```

This works when the dialog first appears, but if I resize the page or control, the dialog does not resize, even after closing and reopening the dialog.

I'm guessing the value is only being read on init.

**Comment by sconix** [comment]
Just use [cpWidth]="'auto'" it should then take the width of the element when opened.

**Comment by errorx666** [comment]
It takes on the width of the element when opened, but if you resize the window, it doesn't resize with it.

**Comment by sconix** [comment]
Dialog should close when you resize, but it seems that it does not. At least thats how popups usually work. I will need to fix that.

**Comment by errorx666** [comment]
Actually, even after dismissing the dialog and reopening it, it seems to keep the size the textbox had the first time it was opened.

**Comment by sconix** [comment]
Or actually not sure how the native popups work  I will check and either add resize or closing.

**Comment by sconix** [comment]
Ok, I will check it and rekease fix tomorrow.

**Comment by sconix** [comment]
This is now fixed in the latest 4.3.2 release.
